internal/godocfx/index.go

```diff
@@ -28,7 +28,7 @@ import (
 
 // indexer gets a limited list of entries from index.golang.org.
 type indexer interface {
-	get(prefix string, since time.Time) (entries []indexEntry, last time.Time, err error)
+	get(prefixes []string, since time.Time) (entries []indexEntry, last time.Time, err error)
```

Comment:
Are overlapping prefixes an issue that may need consideration?

Reply:
No, I don't think so. We won't process duplicate modules because we don't process each prefix independently -- if a module matches any prefix, it's included.